frontend/src/metabase/containers/Unsubscribe.styled.tsx

```diff
@@ -21,15 +21,22 @@ export const LayoutBody = styled.div`
   flex: 1;
 `;
 
-export const LayoutIllustration = styled.div`
+export const LayoutIllustration = styled.div<{
```

Comment:
This is for the unsubscribe page. The style should be the same as `frontend/src/metabase/auth/components/AuthLayout/AuthLayout.styled.tsx` and `frontend/src/metabase/home/components/HomeLayout/HomeLayout.styled.tsx`

To see this page, visit `/unsubscribe?hash=hash&email=email&pulse-id=pulse-id`

![image](https://github.com/metabase/metabase/assets/1937582/43c039de-b4fb-495a-b139-fcc42a081977)